jestconfig.json

```diff
@@ -7,7 +7,7 @@
     "./node_modules/enzyme-to-json/serializer"
   ],
   "moduleNameMapper": {
-    "\\.(svg)$": "<rootDir>/__mocks__/fileMock.js",
+    "\\.(svg|jpg)$": "<rootDir>/__mocks__/fileMock.js",
```

Comment:
Perhaps we should throw on png in here as well, as I do envision other teams will be using that probably sooner than later?

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/855d5fc5c14ce5dc2a29cb07d7d5575af2320962).